pom.xml

```diff
@@ -67,9 +67,9 @@
 
     <properties>
         <gravitee-bom.version>1.0</gravitee-bom.version>
-        <gravitee-common.version>1.16.2</gravitee-common.version>
+        <gravitee-common.version>1.22.0-SNAPSHOT</gravitee-common.version>
```

Comment:
Why are we putting `-SNAPSHOT` versions ?

Reply:
Not sure to understand your question. Just for your information, this PR is part of this issue gravitee-io/issues#5966 which contains multiples PR on different repositories. We put a SNAPSHOT version in order to embed the fix in AM to be able to be properly test it. The SNAPSHOT will be removed once the PRs related to the issue will be merge and the release will be performed.

Reply:
> We put a SNAPSHOT version in order to embed the fix in AM to be able to properly test it.
@jhaeyaert that exactly answers my question. Thank you !